admin/horace_install.m

```diff
@@ -0,0 +1,164 @@
+function horace_install()
+%HORACE_INSTALL install this instance of Horace
+%
+HORACE_ON_PLACEHOLDER = '${Horace_CORE}';
+HERBERT_ON_PLACEHOLDER = '${Herbert_CORE}';
+
+install_root = fileparts(mfilename('fullpath'));
+
+% Find directories containing herbert/horace_init.m
+% Do this first before find_userpath, so we can error out before potentially
+% creating a directory
+hor_init_dir = find_directory('horace_init.m', {fullfile(install_root, 'Horace')});
+her_init_dir = find_directory('herbert_init.m', {fullfile(install_root, 'Herbert')});
+
+% Find/create userpath - this is automatically added to Matlab path on launch
+user_path = find_userpath();
+
+% Find and install horace_on
+horace_on_path = find_file('horace_on.m.template', {install_root});
+install_file( ...
+    horace_on_path, ...
+    fullfile(user_path, 'horace_on.m'), ...
+    {HORACE_ON_PLACEHOLDER, HERBERT_ON_PLACEHOLDER}, ...
+    {hor_init_dir, her_init_dir} ...
+);
+
+% Find and install herbert_on
+herbert_on_path = find_file('herbert_on.m.template', {install_root});
+install_file( ...
+    herbert_on_path, ...
+    fullfile(user_path, 'herbert_on.m'), ...
+    {HERBERT_ON_PLACEHOLDER}, ...
+    {her_init_dir} ...
+);
+
+% Copy worker_v2 script (required by parallel routines) to userpath
+worker_path = find_file('worker_v2.m.template', {install_root});
+install_file(worker_path, fullfile(user_path, 'worker_v2.m'));
+
+disp('Horace successfully installed.')
```

Comment:
Should we run some simple tests (i.e. calling horace_on, herbert_on and a quick parallel init) before claiming it's installed?